corehq/ex-submodules/casexml/apps/phone/xml.py

```diff
@@ -107,3 +107,8 @@ def get_data_element(name, dict):
         sub_el.attrib = {"key": k}
         elem.append(sub_el)
     return elem
+
+
+def synclog_id_from_restore_payload(restore_payload):
```

Comment:
This still seems to only be used in tests. Fine to leave it here but doesn't seem necessary to have it outside of test.